docs/resources/workflow.md

```diff
@@ -17,68 +17,68 @@ A draft version of the public API for workflows.
 
 ### Required
 
-- `condition_groups` (Attributes Set) (see [below for nested schema](#nestedatt--condition_groups))
-- `continue_on_step_error` (Boolean)
-- `expressions` (Attributes Set) (see [below for nested schema](#nestedatt--expressions))
-- `include_private_incidents` (Boolean)
-- `name` (String)
-- `once_for` (List of String)
-- `runs_on_incident_modes` (List of String)
-- `runs_on_incidents` (String)
-- `state` (String)
-- `steps` (Attributes List) (see [below for nested schema](#nestedatt--steps))
-- `trigger` (String)
+- `condition_groups` (Attributes Set) Groups of prerequisite conditions. All conditions in at least one group must be met (see [below for nested schema](#nestedatt--condition_groups))
+- `continue_on_step_error` (Boolean) Whether to continue executing the workflow if a step fails
+- `expressions` (Attributes Set) The expressions to be executed and prepared for steps and conditions (see [below for nested schema](#nestedatt--expressions))
+- `include_private_incidents` (Boolean) Whether to include private incidents
+- `name` (String) The human-readable name of the workflow
+- `once_for` (List of String) This workflow will run 'once for' a list of references
+- `runs_on_incident_modes` (List of String) Incidents in these modes will be affected by the workflow
+- `runs_on_incidents` (String) Which incidents should the workflow be applied to?
```

Comment:
these two are confusingly similar

Reply:
I think if you put in the two options, it's easier to read